source/Calamari.Terraform/TerraformCommand.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Calamari.Commands.Support;
+using Calamari.Deployment;
+using Calamari.Deployment.Conventions;
+using Calamari.Integration.FileSystem;
+using Calamari.Integration.Packages;
+using Calamari.Integration.Processes;
+using Calamari.Integration.Substitutions;
+
+namespace Calamari.Terraform
+{
+    public abstract class TerraformCommand : Command
```

Comment:
Talking to @zentron I think we want to move away from adding a new `*Command` for each capability of Calamari. I'm not certain it's worth holding up this particular PR since the Calamari.exe API isn't something we keep compatible over time.